### PR TITLE
hotfix: Detect file extension in Dataverse file renderer.

### LINKED ISSUE #1463: [Production and Staging] [Dataverse] JPG files don't render from Dataverse Add-on
This file was uploaded to Dataverse and this is how it renders:

screen shot 2015-01-08 at 1 33 50 pm

I'm going to check out some other file types and see if this behavior happens there too.

### PR DESCRIPTION
**Pull request by jmcarp**
Resolves #1463

### CODE REVIEW

**Comment by kmjungersen** [comment]
:+1: :octocat:

File: website/addons/dataverse/views/crud.py [hunk 1]
```diff
@@ -179,7 +180,7 @@ def dataverse_view_file(node_addon, auth, **kwargs):
             node_addon,
             cache_file_name,
             start_render=True,
-            remote_path=file_obj.file_id,
+            remote_path=file_obj.file_id + ext,
```
Comment (kmjungersen) [inline website/addons/dataverse/views/crud.py]: File extensions are nice :)

**Comment by kmjungersen** [comment]
@jmcarp and @sloria: Is this live on production?

**Comment by sloria** [comment]
@kmjungersen Yes.

**Pull request merged**